docker-compose.yml

```diff
@@ -96,6 +96,8 @@ services:
       - SPRING_PROFILES_ACTIVE=endtoend
       - FILESTORAGE_HOST=http://soknadsfillager:9042
       - JOARK_HOST=http://arkiv-mock:8092
+      - SAF_URL=http://arkiv-mock:8092
```

Comment:
Skrive kommentar enten her eller i arkiv-mock at de mockes i arkiv-mock selv om det ikke er mock for JOARK.